Description: Avoid potential privacy breaches in templates
Author: Jonas Smedegaard <dr@jones.dk>
License: GPL-3+
Last-Update: 2018-06-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/data/dzslides/template.html
+++ b/data/dzslides/template.html
@@ -48,7 +48,7 @@
 
 <section>
     <figure> <!-- Figures are used to display images and videos fullpage -->
-      <img src="http://placekitten.com/g/800/600">
+      <img src="cat.png">
       <figcaption>An image</figcaption>
     </figure>
     <div role="note">Kittens are so cute!</div>
@@ -56,7 +56,7 @@
 
 <section>
     <figure> <!-- Videos are automatically played -->
-      <video src="http://videos-cdn.mozilla.net/brand/Mozilla_Firefox_Manifesto_v0.2_640.webm" poster="http://www.mozilla.org/images/about/poster.jpg"></video>
+      <video src="video.webm" poster="video.jpg"></video>
       <figcaption>A video</figcaption>
     </figure>
 </section>
@@ -68,16 +68,13 @@
 <!-- Your Style -->
 <!-- Define the style of your presentation -->
 
-<!-- Maybe a font from http://www.google.com/webfonts ? -->
-<link href='http://fonts.googleapis.com/css?family=Oswald' rel='stylesheet'>
-
 <style>
   html, .view body { background-color: black; counter-reset: slideidx; }
   body, .view section { background-color: white; border-radius: 12px }
   /* A section is a slide. It's size is 800x600, and this will never change */
   section, .view head > title {
       /* The font from Google */
-      font-family: 'Oswald', arial, serif;
+      font-family: 'DejaVu Sans Condensed', 'Liberation Sans', 'Nimbus Sans L', arial, serif;
       font-size: 30px;
   }
 
--- a/data/templates/default.dzslides
+++ b/data/templates/default.dzslides
@@ -31,15 +31,12 @@
   <link rel="stylesheet" href="$css$">
 $endfor$
 $else$
-<link href='https://fonts.googleapis.com/css?family=Oswald' rel='stylesheet'>
-
 <style>
   html, .view body { background-color: black; counter-reset: slideidx; }
   body, .view section { background-color: white; border-radius: 12px }
   /* A section is a slide. It's size is 800x600, and this will never change */
   section, .view head > title {
-      /* The font from Google */
-      font-family: 'Oswald', arial, serif;
+      font-family: 'DejaVu Sans Condensed', 'Liberation Sans', 'Nimbus Sans L', arial, serif;
       font-size: 30px;
   }
 
--- a/data/templates/default.html5
+++ b/data/templates/default.html5
@@ -34,9 +34,6 @@
 $if(math)$
   $math$
 $endif$
-  <!--[if lt IE 9]>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
-  <![endif]-->
 $for(header-includes)$
   $header-includes$
 $endfor$
--- a/data/templates/default.revealjs
+++ b/data/templates/default.revealjs
@@ -237,7 +237,7 @@
 $endif$
 $if(mathjax)$
         math: {
-          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
+          mathjax: '/usr/share/javascript/mathjax/MathJax.js',
           config: 'TeX-AMS_HTML-full',
           tex2jax: {
             inlineMath: [['\\(','\\)']],
--- a/src/Text/Pandoc/Writers/Math.hs
+++ b/src/Text/Pandoc/Writers/Math.hs
@@ -52,7 +52,7 @@
                    InlineMath  -> DisplayInline
 
 defaultMathJaxURL :: String
-defaultMathJaxURL = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/"
+defaultMathJaxURL = "/usr/share/javascript/mathjax/"
 
 defaultKaTeXURL :: String
-defaultKaTeXURL = "https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.8.3/"
+defaultKaTeXURL = "/usr/lib/nodejs/katex/dist/"
